spec/test/sample_schemas/duplicate_names/ncbi/ncbi_taxon.yaml

```diff
@@ -0,0 +1,64 @@
+name: ncbi_taxon
```

Comment:
Next four files are just to check that the validator picks up duplicate file names.